apps/api/.env.example

```
@@ -0,0 +1,13 @@
+# Required
+DATABASE_URL="postgresql://USER:PASSWORD@localhost:5432/your_database_name?schema=public"
+JWT_SECRET="replace-with-a-strong-random-secret"
+
+# Optional (good defaults shown)
+PORT=8080
+CORS_ORIGINS=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CORS_ORIGINS key should go before the PORT key

(UnorderedKey)